### ISSUE TITLE
cbl_flutter:installBinaries

### ISSUE THREAD

**Issue by natgross** [cofu-app/cbl-dart#61]
Using vs_code on Windows 10. Simply can't build a project.
From the doc page it seem like I need the cbl_flutter dependency. Is there any other way to get the binaries and where do I put them?

build output:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':cbl_flutter:installBinaries'.
> A problem occurred starting process 'command './tool/install_binaries.sh''

```

**Comment by blaugold** [comment]
Hi @natgross, I'm sorry you're running into this issue.

You're right, you need `cbl_flutter` to include the binaries in the app build, but support for downloading them has not been implemented yet on Windows.
There really isn't a good/simple workaround at the moment because `cbl_flutter` is a Flutter plugin which does a bit more than just downloading the binaries.

The work to resolve this issue is being tracked in #62.

**Comment by blaugold** [comment]
A fix for `cbl_flutter` has been published in version `0.4.0+1`.
